Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM debian:stable-20170620
```

Comment:
I thought because the release is so [recent](https://www.debian.org/News/2017/20170617) that I wanted to stress the upgrade. If we remove the date tag I guess that `debian:stretch` makes more sense. Years from now they'll move `:stable` to next release, and the build might break.